### ISSUE TITLE
Update Open Graph Thumbnail

### ISSUE THREAD

**Issue by pathumego** [fontforge/fontforge.github.io#72]
Update / Add open graph thumbnail of a new screenshot.

![screen shot 2015-05-25 at 7 14 00 am](https://cloud.githubusercontent.com/assets/6846609/7790701/d4e7ee7c-02ad-11e5-978e-89c51bd71847.png)

**Comment by andreaslarsen** [comment]
![c3324e967b71afa47f94db62f15df203](https://cloud.githubusercontent.com/assets/3859425/7813030/af914924-03b7-11e5-8fcc-cc8769865bb7.png)

Is this too big/detailed?

**Comment by pathumego** [comment]
Yeah May be drop the Open Font dialog, And make the grid on Font Window bigger?

**Comment by andreaslarsen** [comment]
Yeah...let's wait a day or two. Template for needed information here https://github.com/andreaslarsen/animalnoteheads/blob/gh-pages/_includes/head.html
